docs/content/documentation/deployment/github-pages.md

```diff
@@ -14,9 +14,52 @@ so we will focus on the method that will work regardless of the domain.
 
 We can use any continuous integration (CI) server to build and deploy our site. For example:
 
- * [Github Actions](https://github.com/shalzz/zola-deploy-action)
+ * [Github Actions](#github-actions)
  * [Travis CI](#travis-ci)
 
+## Github Actions
+
+Using *Github Actions* for the deployment of your Zola-Page on Github-Pages is pretty easy. You basically need three things:
+
+1. A *Personal access token* to give the *Github Action* the permission to push into your repository.
+2. Create the *Github Action*.
+3. Check the *Github Pages* section in repository settings.
+
+Let's start with the token.
+
+For creating the token either click on [here](https://github.com/settings/tokens) or go to Settings > Developer Settings > Personal access tokens. Under the *Select Scopes* section, give it *repo* permissions and click *Generate token*. Then copy the token, navigate to your repository and add in the Settings tab the *Secret* `TOKEN` and paste your token in it.
+
+Next we need to create the *Github Action*. Here we can make use of the [zola-deploy-action](https://github.com/shalzz/zola-deploy-action). Go to the *Actions* tab of your repository, click on *set up a workflow yourself* to get a blank workflow file. Copy the following script into it and commit it afterwards.
+
+```yaml
+# On every push this script is executed
+on: push
+name: Build and deploy GH Pages
+jobs:
+  build:
+    name: shalzz/zola-deploy-action
+    runs-on: ubuntu-latest
+    steps:
+    # Checkout
+    - uses: actions/checkout@master
+    # Build & deploy
+    - name: shalzz/zola-deploy-action
+      uses: shalzz/zola-deploy-action@master
```

Comment:
Can you use explicit tags instead of master there? Major versions can have breaking changes and using master could break someone site

Reply:
Done.